src/client/components/CryptoList/CryptoList.test.tsx

```diff
@@ -0,0 +1,66 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import { CryptoList } from "@coin-view/client";
+import { mockData } from "@coin-view/mocks";
+import { mockHistoricalData } from "@coin-view/mocks";
+import { CoinListItem, CurrencyType } from "@coin-view/types";
+import { HistoricalDataType } from "@coin-view/client";
+import { I18nextProvider } from "react-i18next";
+import { instanceEN } from "@coin-view/mocks";
+import { AppContext } from "@coin-view/context";
+
+const getHistoricalData = jest.fn();
+const addToFavorites = jest.fn();
+
+const mockCoinListItem: CoinListItem = mockData.data[0];
+
+const mockHistoricalDataType: Record<CurrencyType, HistoricalDataType> = {
+  PLN: mockHistoricalData,
+  USD: mockHistoricalData,
+};
+
+const Wrapper = ({ children }: { children: JSX.Element }) => {
```

Comment:
Usually instead of `JSX.Element`, `React.ReactNode` is used. As long as it works